XenAdmin/Dialogs/PropertiesDialog.cs

```diff
@@ -317,6 +318,12 @@ private void Build()
                         dialog.ShowDialog(Program.MainWindow);
                     }
                 }
+                if (isPoolOrStandalone && Helpers.XapiEqualOrGreater_23_27_0(connection)

```

Comment:
Is `isPoolOrStandalone` correct here? This means that if someone wants to set different settings for only one host in a multi-host pool, they cannot do it.

Reply:
It's correct. Have confirmed with Enzo that NRPE page will be shown for a pool and standalone host, not shown for a host in a multi-host pool.